src/__experimental__/components/tooltip/tooltip.component.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledTooltipInner, StyledTooltipWrapper, StyledTooltipPointer } from './tooltip.style';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+import tagComponent from '../../../utils/helpers/tags';
+
+class Tooltip extends React.Component {
+  static propTypes = {
+    /**
+     * Sets alignment of pointer on tooltip
+     * Options: top, bottom, center, right, left
```

Comment:
list of options will be obtained from `OptionsHelper`, so we no longer need to list them in documentation

Reply:
Yep, sorry thought I'd done this